### PR TITLE
Replace CI with GitHub Actions

### PR DESCRIPTION
**Pull request by alexgaribay**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  push:
+    branches:
+      - master
+
+jobs:
+  mix_test:
+    name: mix test (OTP ${{matrix.otp}} | Elixir ${{matrix.elixir}})
+    strategy:
+      matrix:
+        elixir: ['1.4.5', '1.5.1']
```
Comment (josevalim) [inline .github/workflows/ci.yml]: I believe this value is not being used? I will try removing it!

Comment (josevalim) [reply .github/workflows/ci.yml]: Nope, we need it! :D

Comment (alexgaribay) [reply .github/workflows/ci.yml]: Correct. At moment with GitHub actions, the `includes` section is not additive like Travis. It acts as a modifier for configurations possible from the matrix.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**